src/components/Navbar.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+// eslint-disable-next-line import/no-extraneous-dependencies
```

Comment:
- [ ] As requested of you by the previous reviewer, kindly ensure that you remove the linter disablers from your code base. I have checked run ```npx eslint .``` and the linters all pass. Kindly remove this line.

Reply:
Hi @Nemwel-Boniface 
The requested changes have been made. Thanks.